pom.xml

```diff
@@ -138,6 +132,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
+                <version>1.6.0</version>
```

Comment:
check line 108. There is a `grpc.version`. It would be better if we keep consistent.

Reply:
I don't really get what its relationship is with `grpc.version`...... Do you mean by making this a variable? It seems unnecessary because its version should not be changed frequently.

Reply:
I think it's fine to leave plugin version fixed.
Check google's pom
https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/pom.xml

Reply:
Yeah. I eam to make a veritable and use it here just like `grpc.version`.